templates/projects/js/amp_store.js

```diff
@@ -0,0 +1,18 @@
+<script type="text/javascript">
+$(document).ready(function () {
+  $('#input_data_class').change(function () {
+      if ($('#input_data_class').val().includes('L1')){
```

Comment:
This line could probably be optimized to [reference](https://stackoverflow.com/a/9655728) the changed element using [event.target](https://api.jquery.com/event.target/). For example:

```javascript
 $('#input_data_class').change(function (e) {
      if ($(e.target).val().includes('L1')) { ... }
 });
```

Reply:
Thanks Kory, the comments are fixed in latest commit!